python/mxnet/gluon/parameter.py

```diff
@@ -919,23 +919,49 @@ def load(self, filename, ctx=None, allow_missing=False,
                 assert name.startswith(restore_prefix), \
                     "restore_prefix is '%s' but Parameters name '%s' does not start " \
                     "with '%s'"%(restore_prefix, name, restore_prefix)
-        lprefix = len(restore_prefix)
         ndarray_load = ndarray.load(filename)
+        self.load_dict(ndarray_load, ctx, allow_missing,
+                       ignore_extra, restore_prefix, filename, cast_dtype)
+
+    def load_dict(self, param_dict, ctx=None, allow_missing=False,
```

Comment:
Is the change related to this PR?

Reply:
yes it is related to the PR, it is used in the convert_hybrid_block